src/renderer/components/ui/activities/begin/games/MyCracha.vue

```diff
@@ -43,104 +63,150 @@ export default {
     components: {
         ...FormProps,
     },
-    mixins: [MapMixins, ListMixin, CreateAnswersMixins,],
+    mixins: [MapMixins, ListMixin, CreateAnswersMixins],
     data() {
         return {
-            userName: null
+            userName: null,
+            hasImage: false,
+            hasNoImage: false,
         }
     },
     computed: {
         name() {
-            return this.userName ? this.userName : {text: 'Fulano'}
+            return this.userName ? this.userName : { text: 'Fulano' }
         },
-        imgSrc(){
+        imgSrc() {
             return this.userName && this.userName.imgSrc ? this.userName.imgSrc : ''
-        }
+        },
     },
     mounted() {
         this.createAnswersArray()
-        if (this.userName === null)
-            this.userName = this.getKeys[0]
+        if (this.userName === null) this.userName = this.getKeys[0]
     },
     methods: {
         ...mapActions('Activity', ['triggerSuccess']),
         handleFileUpload(index) {
+            this.hasNoImage = false
             let file = this.$refs[`file${index}`].files[0]
             let reader = new FileReader()
 
-            reader.onload = e => {
-                Vue.set(this.userName, 'imgSrc',e.target.result)
+            reader.onload = (e) => {
+                Vue.set(this.userName, 'imgSrc', e.target.result)
                 //adicionando o background liso após a foto ser carregada
                 this.$refs['imgContainer'].classList.add('grey-bg')
-                setTimeout(this.triggerSuccess, 400)
+                this.hasImage = true
             }
-            reader.onerror = function(error) {
+            reader.onerror = function (error) {
                 console.log(error)
             }
-            if (file)
-                reader.readAsDataURL(file)
+            if (file) reader.readAsDataURL(file)
+        },
+        continueGame() {
+            if (this.hasImage) {
+                setTimeout(this.triggerSuccess, 400)
+            } else {
+                this.hasNoImage = true
+            }
```

Comment:
Não é para passar quando não há imagem.
```suggestion
            if (this.hasImage) setTimeout(this.triggerSuccess, 400)
```